metadata/multi-omics.yaml

```diff
@@ -0,0 +1,13 @@
+---
+name: "multi-omics"
+type: "use"
+topic_type: field
+title: "Multiomics"
+summary: "Set of tutorials for tools which process or analyse multiomics data."
+
+tag_based: true
+
+gitter: galaxy-multi-omics:matrix.org
```

Comment:
This is different from the one above, and I think both do not exist ...
I'm also not sure if we should really create a separate room for it or if we should reuse one of the other rooms